src/System.Text.JsonLab/System/Text/Json/Utf8JsonReader.cs

```diff
@@ -197,9 +185,7 @@ private void StartObject()
             if (Depth > MaxDepth)
                 ThrowJsonReaderException(ref this, ExceptionResource.ObjectDepthTooLarge);
 
-#if UseInstrumented
             _position++;
```

Comment:
Why do you always increment the position now?

Reply:
> I did not avoid incrementing `_position` in all places (when Instrument is false), since an if check is likely slower than just incrementing a field.

Alternatively, I would have `if (Instrumented) _position++;` everywhere. Essentially, if instrumented is false, the value `_position` is meaningless. Should I add if checks around all `_position`  increments?